lib/Wikia/src/Service/Helios/HeliosClientImpl.php

```diff
@@ -13,6 +13,8 @@ class HeliosClientImpl implements HeliosClient
 	const CLIENT_ID = "client_id";
 	const CLIENT_SECRET = "client_secret";
 
+	const HELIOS_AUTH_HEADER = "X-Wikia-UserId";
```

Comment:
@drsnyder, @nmonterroso: Added `Wikia\Service\Constants` and moved the `X-Wkia-UserId` there (a8cd822).